apps/src/templates/projects/projectConstants.ts

```diff
@@ -83,4 +83,6 @@ export const PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE: {
 } = {
   music: '/shared/images/fill-70x70/courses/logo_music.png',
   pythonlab: '/shared/images/fill-70x70/courses/logo_pythonlab.png',
+  // Temporary placeholder
+  music_dance_ai: '/blockly/media/dance/placeholder.png',
```

Comment:
@dju90 could use your input here: this key (which is used in a variety of places) will ultimately be in the user-facing URL (`projects/music_dance_ai/<channel>`), as well as the project type key that the RED team will use to look up metrics. Let me know what a good path name might be (we can discuss in Slack as well).